docs/security-champion/index.md

```diff
@@ -1,13 +1,11 @@
 # Security Champion
 
-!!! warning
-    The Security Champion initiative has yet not been *officially* launched in Equinor, so everything on this site is as of now work in progress. We intend to have a kickoff in Q3 2022, so stay tuned on these pages.
+## Info about Security Champion
 
-## Info about security champion
-
-Ideally, each Equinor delivery team would have one or more team-member who takes on the role of Security Champion.
+Ideally, each DevOps team in Equinor should have one or more team-member who takes on the role of Security Champion.
```

Comment:
Maybe it should say all teams developing software in Equinor? As not everyone operates their software, some just write libraries etc.

Reply:
fixed